### PR TITLE
Add http config, auth service to frontend;

### PR DESCRIPTION
**Pull request by IvaskevychYuriy**
Add http config, auth service to frontend;
Fix api endpoints for home controller

### CODE REVIEW

File: SimpleDrive.App/ClientApp/services/authentication.service.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import http from "./core/http";
+import { AxiosResponse } from "axios";
+
+import { LoginInfo } from "../models/login-model";
+import { UserProfile } from "../models/user-profile";
+
+class AuthenticationService {
+    private static currentUserKey = 'currentUser';
```
Comment (nazar554) [inline SimpleDrive.App/ClientApp/services/authentication.service.ts]: `public static readonly`
`CURRENT_USER_KEY`

File: SimpleDrive.App/ClientApp/services/authentication.service.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    private _userProfile: UserProfile;
```
Comment (nazar554) [inline SimpleDrive.App/ClientApp/services/authentication.service.ts]: `userProfile`

File: SimpleDrive.App/ClientApp/components/FetchData.tsx [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router';
-import 'isomorphic-fetch';
+import authService from '../services/authentication.service';
```
Comment (nazar554) [inline SimpleDrive.App/ClientApp/components/FetchData.tsx]: `CamelCase`

**Review by nazar554 — CHANGES_REQUESTED** [review]

File: SimpleDrive.App/ClientApp/components/FetchData.tsx (hunk 3 above)
Comment (nazar554) [reply SimpleDrive.App/ClientApp/components/FetchData.tsx]: `AuthenticationService.ts`

File: SimpleDrive.App/ClientApp/services/authentication.service.ts (hunk 1 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/ClientApp/services/authentication.service.ts]: Fixed in https://github.com/SimpleDriveOrganization/SimpleDrive/pull/14/commits/ed82b6545cfea2abe971dd85811ed9519b10e325

File: SimpleDrive.App/ClientApp/components/FetchData.tsx (hunk 3 above)
Comment (IvaskevychYuriy) [reply SimpleDrive.App/ClientApp/components/FetchData.tsx]: CamelCase is useless here - singleton is exported as service

Fixed in https://github.com/SimpleDriveOrganization/SimpleDrive/pull/14/commits/ed82b6545cfea2abe971dd85811ed9519b10e325

**Pull request merged**